uint32 seq
float64[] position_x
float64[] position_y
float64[] position_yaw
float64[] target_x
float64[] target_y